yarn.lock

```diff
@@ -2616,13 +2616,13 @@ __metadata:
   languageName: node
   linkType: hard
 
-"@react-native-community/blur@npm:4.4.1":
-  version: 4.4.1
-  resolution: "@react-native-community/blur@npm:4.4.1"
+"@react-native-community/blur@npm:4.3.0":
```

Comment:
You should yarn again after the revert so this is removed.

Reply:
Done